code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -1,26 +1,28 @@
+/// Adds a newline to the examine list if the above entry is not empty and is not the first element in the list
+#define ADD_NEWLINE_IF_NECESSARY(list) if(list && list[length(list)] && length(list) > 1) { list += "" }
+
 /mob/living/carbon/human/examine(mob/user)
-//this is very slightly better than it was because you can use it more places. still can't do \his[src] though.
-	var/t_He = p_They()
-	var/t_His = p_Their()
-	var/t_his = p_their()
-	var/t_him = p_them()
-	var/t_has = p_have()
-	var/t_is = p_are()
-	var/obscure_name
-	var/obscure_examine
+	var/obscure_name = FALSE
+	var/obscure_examine = FALSE
 
 	if(isliving(user))
-		var/mob/living/L = user
-		if(HAS_TRAIT(L, TRAIT_PROSOPAGNOSIA) || HAS_TRAIT(L, TRAIT_INVISIBLE_MAN))
+		if(HAS_TRAIT(user, TRAIT_PROSOPAGNOSIA) || HAS_TRAIT(user, TRAIT_INVISIBLE_MAN))
```

Comment:
I'm fairly sure TRAIT_INVISIBLE_MAN is meant to be checking the person observed and not the observer, right?

Reply:
That seems sensible

Reply:
```suggestion
		if(HAS_TRAIT(user, TRAIT_PROSOPAGNOSIA) || HAS_TRAIT(src, TRAIT_INVISIBLE_MAN))
```